src/main/java/com/alkemy/ong/entity/CategoryEntity.java

```diff
@@ -11,7 +11,6 @@
 @Entity
 @Getter @Setter @ToString @RequiredArgsConstructor
 @SQLDelete(sql="UPDATE categorys SET softDelete = true WHERE id=?")
```

Comment:
I suggest that the table name changes for categories in stead of categorys (just a spelling thing) but we have to consider dropping our DB

Reply:
I think the name complies with what is stated on the ticket. It's fine with me to leave it like that.